chaoscenter/graphql/server/server.go

```diff
@@ -116,8 +118,13 @@ func main() {
 		},
 	})
 
-	// to be removed in production
-	srv.Use(extension.Introspection{})
+	enableIntrospection, err := strconv.ParseBool(utils.Config.EnableGQLIntrospection)
+	if err != nil {
+		logrus.Errorf("unable to parse boolean value %v", err)
+	}
+	if err == nil && enableIntrospection == true {
```

Comment:
Do we need nil check here? Already check above

Reply:
since line 125 is not in else, even after err is not nil control will still come to line 125 as I am not returning the err in line in 123

Reply:
Okay got it!

Reply:
else-if can help here